drizzlepac/test_config_utils.py

```diff
@@ -12,19 +12,26 @@
 
```

Comment:
This file needs to be renamed to avoid being picked up as a pytest test in the CI environments.